src/main/java/in/oogway/library/ingestor/Ingestor.java

```diff
@@ -0,0 +1,71 @@
+package in.oogway.library.ingestor;
+
+import com.esotericsoftware.yamlbeans.YamlException;
+import com.esotericsoftware.yamlbeans.YamlReader;
+import in.oogway.library.storage.LocalStorage;
+import in.oogway.runner.transformer.Transformer;
+import org.apache.commons.io.input.CharSequenceReader;
+import org.apache.log4j.Logger;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.util.Map;
+
+public class Ingestor implements LocalStorage {
+
+    public static final org.apache.log4j.Logger LOGGER = org.apache.log4j.Logger.getLogger(Logger.class);
+
```

Comment:
execute(String path) {
    Map content = loadContent(path);
    String sourceId = yamlMap.get("source").toString();
    Map source = loadContent(sourceId);
    // print source as of now. Will add stubs later.
    
    String transformationId = yamlMap.get("transformation").toString();
    Map tf = loadContent(transformationId);
    ArrayList<Transformer> transformers = inflateTransformers(tf["stages"]);
    
    Dataframe df = source.Get();
    
    for (transformers: t {
       df = t.Run(df);
    })

   sink.Write(df);
}